src/components/store/Diamonds/index.tsx

```diff
@@ -9,7 +10,8 @@ interface DiamondsProps {
 
 const Diamonds = ({ count, className }: DiamondsProps) => (
   <span className={className}>
-    {numberFormat.format(count)} <span className={styles.diamond}>diamonds</span>
+    {(count >= 1e6 ? compactFormat : numberFormat).format(count)}{' '}
```

Comment:
```suggestion
    {(count >= 1e5 ? compactFormat : numberFormat).format(count)}{' '}
```
Since a million is a really high limit that I don't think anyone could possibly reach in their ACM lifetime, I think formatting above 50k or 100k might make more sense given the most expensive items are around 15k so the precision isn't as relevant anymore to determine if they can afford something.